tests/apps/test_measure.py

```diff
@@ -271,6 +271,11 @@ def test_with_orig(get_orig, measure, storer):
 
     # configure max calls originated to length of of storer buffer
     cdr_storer = orig.measurers['CDR'].storer
+
+    if (cdr_storer.storetype.__name__ == 'CSVStore'
```

Comment:
Not sure what's up here but we're skipping this measurement test for now since it's flaky in github actions.

Reply:
This was to skip that storertype, but not skip the other storertype. I didn't know if I should skip the test completely for all python 3.7+ or not.